frontend/public/components/_edit-yaml.scss

```diff
@@ -51,5 +51,5 @@
 
 // This can't be customized in a theme, so use CSS.
 .yaml-editor .monaco-editor .margin .current-line {
-  background-color: var(--pf-global--BackgroundColor--dark-200);
+  background-color: var(--pf-global--BackgroundColor--light-300);
```

Comment:
@jhadvig Can you check if this was a problem prior to the PF upgrade? If not, we should try to use the same colors as before.

Reply:
Bgcolor + color of the font prior of the PF upgrade:
![Screenshot 2020-07-16 at 11 25 04](https://user-images.githubusercontent.com/1668218/87654443-34321100-c757-11ea-924f-cb05bad1361d.png)

```css
// Taken from _edit-yaml.scss:48
.yaml-editor .monaco-editor .margin .current-line {
    background-color: var(--pf-global--BackgroundColor--dark-200);
}
// Taken from <style>
.monaco-editor .current-line ~ .line-numbers {
    color: #ffffff;
}
```

Styling in the 4.5 (styling is the same):
<img width="625" alt="Screenshot 2020-07-16 at 11 31 37" src="https://user-images.githubusercontent.com/1668218/87655014-f84b7b80-c757-11ea-8f24-1324967b3c7b.png">